ultet/src/main/scala/za/co/absa/ultet/dbitems/DBTable.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2023 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.ultet.dbitems
+
+case class DBTable (
+                   tableName: String,
+                   description: Option[String],
+                   primaryDBName: String,
+                   owner: String,
+                   columns: Seq[DBTableColumn] = Seq.empty,
```

Comment:
let's write a check that the column can't be there twice or more and that this seq is not empty

Reply:
some checks needed for other fields as well

Reply:
Yup, in later stages.